src/libraries/Common/src/System/Security/Cryptography/CompositeMLDsa.cs

```diff
@@ -0,0 +1,1800 @@
+        ///     represented by this instance.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     The base-64 decoded contents of the PEM text from <paramref name="source" />
+        ///     represent the key in a format that is not supported.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     An error occurred while importing the key.
+        ///   </para>
+        /// </exception>
+        /// <exception cref="PlatformNotSupportedException">
+        ///   The platform does not support Composite ML-DSA. Callers can use the <see cref="IsSupported" /> property
+        ///   to determine if the platform supports Composite ML-DSA.
+        /// </exception>
+        /// <remarks>
+        ///   <para>
+        ///     Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels
+        ///     are found, an exception is thrown to prevent importing a key when
+        ///     the key is ambiguous.
+        ///   </para>
+        ///   <para>This method supports the <c>ENCRYPTED PRIVATE KEY</c> PEM label.</para>
+        /// </remarks>
+        public static CompositeMLDsa ImportFromEncryptedPem(ReadOnlySpan<char> source, ReadOnlySpan<byte> passwordBytes)
+        {
+            ThrowIfNotSupported();
+
+            return PemKeyHelpers.ImportEncryptedFactoryPem<CompositeMLDsa, byte>(
+                source,
+                passwordBytes,
+                ImportEncryptedPkcs8PrivateKey);
+        }
+
+        /// <inheritdoc cref="ImportFromPem(ReadOnlySpan{char})" />
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="source" /> is <see langword="null" />.
+        /// </exception>
+        public static CompositeMLDsa ImportFromPem(string source)
+        {
+            ArgumentNullException.ThrowIfNull(source);
+            ThrowIfNotSupported();
+
+            return ImportFromPem(source.AsSpan());
+        }
+
+        /// <summary>
+        ///   Imports a Composite ML-DSA key from an RFC 7468 PEM-encoded string.
+        /// </summary>
+        /// <param name="source">
+        ///   The text of the PEM key to import.
+        /// </param>
+        /// <returns>
+        ///   The imported Composite ML-DSA key.
+        /// </returns>
+        /// <exception cref="ArgumentException">
+        ///   <para><paramref name="source" /> contains an encrypted PEM-encoded key.</para>
+        ///   <para>-or-</para>
+        ///   <para><paramref name="source" /> contains multiple PEM-encoded Composite ML-DSA keys.</para>
+        ///   <para>-or-</para>
+        ///   <para><paramref name="source" /> contains no PEM-encoded Composite ML-DSA keys.</para>
+        /// </exception>
+        /// <exception cref="CryptographicException">
+        ///   An error occurred while importing the key.
+        /// </exception>
+        /// <exception cref="PlatformNotSupportedException">
```

Comment:
What exception are you intending for "I support composites... but not _this_ composite."?

Reply:
Precedence says CryptoException - when an ArgumentException would be thrown from the underlying key import, we instead throw a CryptoException from ImportFromPem. Updated xmldoc.